src/Elemental/Form/Validate.elm

```diff
@@ -34,6 +35,11 @@ validate (Validator getErrors) input =
 -- CONSTRUCTING VALIDATORS
 
 
+new : (input -> List error) -> Validator error input
```

Comment:
@dhruvio What's this for?

Reply:
Spoke on the phone — commenting here for posterity. Adding an opaque value constructor for use with non-string input values.